app/src/main/java/org/schabi/newpipe/local/playlist/LocalPlaylistFragment.java

```diff
@@ -403,10 +403,24 @@ private void changeThumbnailUrl(final String thumbnailUrl) {
         disposables.add(disposable);
     }
 
+    private void updateThumbnailUrl() {
+        String newThumbnailUrl;
+
+        if(!itemListAdapter.getItemsList().isEmpty()){
+            newThumbnailUrl = ((PlaylistStreamEntry)itemListAdapter.getItemsList().get(0)).thumbnailUrl;
+        }
+        else newThumbnailUrl = "https://i.ytimg.com/";
```

Comment:
Are we legally allowed to use this image? We need to check its license first. Despite the license issue, the image should be a local drawable as @Stypox pointed out correctly.